opencda/core/ml_libs/rl/simulators/__init__.py

```diff
@@ -0,0 +1,9 @@
+'''
+Copyright 2021 OpenDILab. All Rights Reserved:
+Description:
+'''
+from opencda.core.ml_libs.rl import SIMULATORS
+
+if 'carla' in SIMULATORS:
```

Comment:
If our tools only support CARLA, then maybe consider deleting the `SIMULATORS` dependent logics for simplicity.